pkgs/by-name/um/umka/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "umka";
+  version = "1.5.4";
+
+  src = fetchFromGitHub {
+    owner = "vtereshkov";
+    repo = "umka-lang";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-UerEmJdD0/Hx/Pqw3NI3cZwjkX9lRWqI5rL0GGYKFwc=";
+  };
+
+  outputs = [
+    "out"
+    "dev"
+    "lib"
+  ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail "RANLIB                = libtool -static -o" "RANLIB = ar -cru"
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -D build/umka $out/bin/umka
+
+    install -D build/include/umka_api.h $dev/include/umka_api.h
+    install -D build/libumka.a $dev/lib/libumka.a
+
+    install -D build/libumka${stdenv.hostPlatform.extensions.sharedLibrary} \
+      $lib/lib/libumka${stdenv.hostPlatform.extensions.sharedLibrary}
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Statically typed embeddable scripting language";
+    sourceProvenance = with sourceTypes; [ fromSource ];
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ f64u ];
+    platforms = platforms.all;
+    mainProgram = "umka";
+  };
```

```suggestion
  meta = {
    description = "Statically typed embeddable scripting language";
    license = lib.licenses.bsd2;
    maintainers = with lib.maintainers; [ f64u ];
    mainProgram = "umka";
  };
```

Source type fromSource is default so not needed.
Platforms.all is also default

---

Sorry I missed that. Implemented.